pkg/dashboard/resource/apigateway.go

```diff
@@ -158,7 +160,7 @@ func (agr *apiGatewayResource) Create(request *http.Request) (id string, attribu
 func (agr *apiGatewayResource) updateAPIGateway(request *http.Request) (*restful.CustomRouteFuncResponse, error) {
 
 	// get api gateway config and status from body
-	apiGatewayInfo, err := agr.getAPIGatewayInfoFromRequest(request, true, false)
+	apiGatewayInfo, err := agr.getAPIGatewayInfoFromRequest(request, false)
```

Comment:
On this case I decided to keep the boolean, because it is used for validations on the deeper function (on order to not split the validations for this change)

But noted for next time 👍

Reply:
Alright, refactor pr is welcome 😉